service/resource/legacyv1/domains.go

```diff
@@ -1,58 +0,0 @@
-package legacyv1
```

Comment:
Why is the legacy resource vanishing now? This should actually stay until all existing guest clusters are not using it anymore. I am not aware that all AWS CROs have a version bundle now. Was there a migration?

Reply:
This only removes `legacyv1` which uses TPRs. The `legacyv2` resource uses CRDs and is what is in use. The resource router defaults to `legacyv2` if the version bundle is not set so no migration is required.

Reply:
I see. That makes sense. Thanks.